SCClassLibrary/Common/GUI/tools/ServerMeter.sc

```diff
@@ -263,4 +263,11 @@ ServerMeter {
 	isClosed {
 		^window.isClosed
 	}
+
+	position_ { |point|
+		point !? {
```

Comment:
why do you not want to catch an error if someone tries to set the point to nil?

Reply:
I initially believed it would be more user-friendly for the default position, which has been in use for a long time, to be applied automatically. However, my reasoning was not entirely logical. The default position is already invoked within `.meter`, meaning that the `point` argument in the `.position` method does not need to set the default position again. Instead, the user should explicitly define a point when using `.position_`.  

I truly appreciate your guidance in helping me gain a clearer understanding. Thank you! 😊

Done in https://github.com/supercollider/supercollider/pull/6283/commits/f4b583e070b7af46a4346390abae65532bc7ea40